A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_PerformancePage.js

```diff
@@ -180,16 +180,16 @@ class CDUPerformancePage {
         }
 
         // transition altitude - remains editable during take off
-        let transAltCell = "";
+        let transAltCell = "{cyan}[  ]{end}";
```

Comment:
This still needs non-breaking-spaces:

```suggestion
        let transAltCell = "{cyan}[\xa0\xa0\xa0]{end}";
```

You also need to count carefully when looking at references. The reference has 3 whitespaces between the brackets.